### PR TITLE
Add documentation related to the rasa helm chart

### PR DESCRIPTION
**Pull request by tczekajlo**
**Proposed changes**:
- Add documentation related to the rasa helm chart

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by tczekajlo** [comment]
@TyDunn @erohmensing I've added documentation related to the new helm chart for rasa open source. 

Please share your thoughts if you have a minute 🙏🏻  (as usual, I had a hard time with what name I should use, so you can find a mix of `Rasa Assistant/Rasa Open Source/...`) 🙈

**Comment by tczekajlo** [comment]
cc @tmbo

File: docs/docs/how-to-deploy.mdx [hunk 1]
```diff
@@ -53,6 +53,275 @@ However, you can also customize the Helm charts if you have specific requirement
...
+   [Helm CLI](https://helm.sh/docs/intro/install/).
+
+   In case you are using a version `<3.5` of Helm, please update to Helm version
+   `>=3.5`.
+
+### Installation
+
+#### 1. Create Namespace
+
+We recommend installing Rasa Assistant in a separate
```
Comment (TyDunn) [inline docs/docs/how-to-deploy.mdx:167]:
```suggestion
We recommend installing the Rasa Open Source server in a separate
```

Comment (TyDunn) [reply docs/docs/how-to-deploy.mdx:167]: Or maybe just "Rasa Open Source" (without server?)

**Review by TyDunn — CHANGES_REQUESTED** [review]
Great work! We should revisit how / what we recommend re: deployment in the near term, but I think adding this as another option is the best way to proceed for now

File: docs/docs/how-to-deploy.mdx (hunk 1 above)
Comment (tczekajlo) [reply docs/docs/how-to-deploy.mdx:167]: "Rasa Open Source" sounds good to me 😄

File: docs/docs/how-to-deploy.mdx [hunk 2]
```diff
@@ -53,6 +53,275 @@ However, you can also customize the Helm charts if you have specific requirement
...
+#### 2. Create Values File
+
+Prepare an empty file called `rasa-values.yml` which will include all your custom
+configuration for the installation with Helm.
+
+All available values you can find in [the rasa helm chart repository](https://github.com/RasaHQ/helm-charts/tree/main/charts/rasa#values).
+
+:::note
+The default configuration of the Rasa chart deploys a Rasa Open Source Server, downloads a model, and serves the downloaded model.
+Visit [the rasa helm chart repository](https://github.com/RasaHQ/helm-charts/tree/main/charts/rasa#quick-start) to check out more examples of configuration.
```
Comment (erohmensing) [inline docs/docs/how-to-deploy.mdx:198]: Let's decide consistently if we wnat to capitalize `rasa` in this
```suggestion
All available values you can find in [the Rasa helm chart repository](https://github.com/RasaHQ/helm-charts/tree/main/charts/rasa#values).

:::note
The default configuration of the Rasa chart deploys a Rasa Open Source Server, downloads a model, and serves the downloaded model.
Visit [the Rasa helm chart repository](https://github.com/RasaHQ/helm-charts/tree/main/charts/rasa#quick-start) to check out more examples of configuration.
```

File: docs/docs/how-to-deploy.mdx [hunk 3]
```diff
@@ -53,6 +53,275 @@ However, you can also customize the Helm charts if you have specific requirement
...
+    initialModel: "https://github.com/RasaHQ/rasa-x-demo/blob/master/models/model.tar.gz?raw=true"
+  ```
+:::note
+The URL for the initial model download has to point to a tar.gz file and must not require authentication.
+
+:::
+
+If you want to train an initial model you can do this by setting the `applicationSettings.trainInitialModel` to `true`.
+It creates a init container that trains a model based on data located in the `/app` directory. If the `/app` directory is empty it creates a new project.
+[Here](https://github.com/RasaHQ/helm-charts/blob/main/examples/rasa/train-model-helmfile.yaml) you can find an example that shows how to download data files from a git repository and train an initial model.
```
Comment (erohmensing) [inline docs/docs/how-to-deploy.mdx:221]:
```suggestion
You can find an example that shows how to download data files from a git repository and train an initial model in the Rasa helm charts [examples](https://github.com/RasaHQ/helm-charts/blob/main/examples/rasa/train-model-helmfile.yaml).
```

File: docs/docs/how-to-deploy.mdx [hunk 4]
```diff
@@ -53,6 +53,275 @@ However, you can also customize the Helm charts if you have specific requirement
...
+  </TabItem>
+</Tabs>
+
+You can then access the deployment on `http://127.0.0.1:${SERVICE_PORT}`
+
+Visit [the Rasa helm chart README](https://github.com/RasaHQ/helm-charts/tree/main/charts/rasa#exposing-the-rasa-deployment-to-the-public) to learn other ways to expose your deployment.
+
+#### Next Steps
+
+Visit [the Rasa helm chart repository](https://github.com/RasaHQ/helm-charts/tree/main/charts/rasa) where you can find examples of configuration and learn how to e.g. integrate Rasa Open Source deployment with Rasa X.
```
Comment (erohmensing) [inline docs/docs/how-to-deploy.mdx:323]:
```suggestion
Visit [the Rasa helm chart repository](https://github.com/RasaHQ/helm-charts/tree/main/charts/rasa) where you can find examples of configuration and learn how to e.g. integrate your Rasa Open Source deployment with Rasa X.
```

File: docs/docs/how-to-deploy.mdx [hunk 5]
```diff
@@ -53,6 +53,275 @@ However, you can also customize the Helm charts if you have specific requirement
...
+Please
+[create an issue](https://github.com/RasaHQ/helm-charts/issues/new) in this
+repository if you discover bugs or have suggestions for improvements.
+
+:::
+
+
+### Installation Requirements
+
+1. Please check that you installed the Kubernetes or OpenShift command line
```
Comment (erohmensing) [inline docs/docs/how-to-deploy.mdx:84]:
```suggestion
1. Check that you have installed the Kubernetes or OpenShift command line
```

File: docs/docs/how-to-deploy.mdx [hunk 6]
```diff
@@ -53,6 +53,275 @@ However, you can also customize the Helm charts if you have specific requirement
...
+     ```
+
+     </TabItem>
+   </Tabs>
+
+   If you get an error when executing the command, you are not connected to your
+   cluster. To get the command to connect to the cluster please consult your cluster’s
+   admin or the documentation of your cloud provider.
+
+3. Please make sure you have the [Helm CLI](https://helm.sh/docs/intro/install/)
```
Comment (erohmensing) [inline docs/docs/how-to-deploy.mdx:147]:
```suggestion
3. Make sure you have the [Helm CLI](https://helm.sh/docs/intro/install/)
```

**Review by erohmensing — COMMENTED** [review]
Not much to add, looks good 🏋🏽 I'll let Ty approve since he req'd changes

File: docs/docs/how-to-deploy.mdx [hunk 7]
```diff
@@ -53,6 +53,275 @@ However, you can also customize the Helm charts if you have specific requirement
 * Custom: Read the above, as well as the [Advanced Configuration](https://rasa.com/docs/rasa-x/installation-and-setup/customize/#helm-chart)
   documentation, and customize the [open source Helm charts](https://github.com/RasaHQ/rasa-x-helm) to your needs.
 
+## Deploying a Rasa Open Source Assistant
+
+While the above deployment methods involve deploying an assistant with Rasa X, the following instructions describe how to deploy a Rasa Open Source server only
+by using [the rasa helm chart](https://github.com/RasaHQ/helm-charts/tree/main/charts/rasa) in a scalable cluster environment using OpenShift or Kubernetes (K8S).
```
Comment (TyDunn) [inline docs/docs/how-to-deploy.mdx:59]:
```suggestion
by using the [Rasa Helm Chart](https://github.com/RasaHQ/helm-charts/tree/main/charts/rasa) in a scalable cluster environment using OpenShift or Kubernetes (K8S).
```

File: docs/docs/how-to-deploy.mdx [hunk 8]
```diff
@@ -53,6 +53,275 @@ However, you can also customize the Helm charts if you have specific requirement
...
+[Kubernetes cluster](https://kubernetes.io/) or [OpenShift cluster](https://www.openshift.com/).
+Setting up a Kubernetes / OpenShift cluster can be tedious, hence we
+recommend to get a managed cluster from a cloud provider like
+[Google Cloud](https://cloud.google.com/kubernetes-engine),
+[DigitalOcean](https://www.digitalocean.com/products/kubernetes/),
+[Microsoft Azure](https://azure.microsoft.com/en-us/services/kubernetes-service/), or
+[Amazon EKS](https://aws.amazon.com/eks/).
+
+:::note
+The Rasa Helm chart is open-source and available in the
```
Comment (TyDunn) [inline docs/docs/how-to-deploy.mdx:73]:
```suggestion
The Rasa Helm chart is open source and available in the
```

**Review by TyDunn — APPROVED** [review]
Some small tweaks needed based on comments from Ella and me but LGTM! 🚀

**Pull request merged**